samples/samples/connection_sample_test.py

```diff
@@ -0,0 +1,82 @@
+import uuid
+
+from google.cloud import spanner
+import pytest
+
+import connection_sample
+
+
+def unique_instance_id():
+    """ Creates a unique id for the database. """
+    return f"test-instance-{uuid.uuid4().hex[:10]}"
+
+
+def unique_database_id():
+    """ Creates a unique id for the database. """
+    return f"test-db-{uuid.uuid4().hex[:10]}"
+
+
+INSTANCE_ID = unique_instance_id()
+DATABASE_ID = unique_database_id()
+
+
+@pytest.fixture(scope="module")
+def spanner_instance():
+    spanner_client = spanner.Client()
+    instance_config = "{}/instanceConfigs/{}".format(
+        spanner_client.project_name, "regional-us-central1"
+    )
```

Comment:
I don't think you need configurations for instance here

Reply:
It doesn't work without configuration field in constructor.
```
E           grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
E               status = StatusCode.INVALID_ARGUMENT
E               details = "Invalid CreateInstance request."
E               debug_error_string = "{"created":"@1603971166.856155000","description":"Error received from peer ipv4:64.233.165.95:443","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Invalid CreateInstance request.","grpc_status":3}"
E           >
```